userspace/libsinsp/sinsp_filtercheck_event.cpp

```diff
@@ -548,9 +548,6 @@ int32_t sinsp_filter_check_event::parse_field_name(std::string_view val,
 		m_field = &m_customfield;
 
 		res = extract_arg("evt.rawarg", val, &m_arginfo);
-
-		m_customfield.m_type = m_arginfo->type;
```

Comment:
Instead of storing a random arginfo type and print format, store them when we fetch them from the being-extracted event.